cmd/mpd-mpris/main.go

```diff
@@ -18,14 +20,24 @@ var (
 )
 
 func init() {
-	flag.StringVar(&addr, "host", "localhost", "The MPD host.")
+	flag.StringVar(&addr, "host", "", "The MPD host (default localhost)")
 	flag.IntVar(&port, "port", 6600, "The MPD port")
 	flag.StringVar(&password, "pwd", "", "The MPD connection password. Leave empty for none.")
 	flag.BoolVar(&noInstance, "no-instance", false, "Set the MPDris's interface as 'org.mpris.MediaPlayer2.mpd' instead of 'org.mpris.MediaPlayer2.mpd.instance#'")
 }
 
 func main() {
 	flag.Parse()
+	if len(addr) == 0 {
+		env_host := os.Getenv("MPD_HOST")
+		if len(env_host) == 0 {
+			addr = "localhost"
+		} else {
+			addr_pwd := strings.Split(env_host, "@")
```

Comment:
What happens here if a password is not provided? I think this case must be handled separately (since there is now no `@` involved, and `addr` is now the `0th` field.

Reply:
Agree, pushed a fix.